parse-task-issue/util.js

```diff
@@ -0,0 +1,25 @@
+const jsyaml = require('js-yaml');
+const fs = require('fs');
+
+
+function getYMLFileContent(issue) {
+  let start = issue.body.indexOf("```")+3
```

Comment:
instead of adding three, would it be more clear to get the index of `"```yml"`

Reply:
Why do you say yml? the template has taskName after ```

Reply:
to get code highlighting in md you can add the language after the ticks, so the ticks should be ````yml`, is the +3 just to get to the end of the string? maybe just add a comment?

Reply:
I changed this to check the occurrence of `taskName` for start